src/main/java/liquibase/ext/spanner/sqlgenerator/AlterSequenceGeneratorSpanner.java

```diff
@@ -0,0 +1,33 @@
+package liquibase.ext.spanner.sqlgenerator;
```

Comment:
nit (here and elsewhere): add copyright header

Reply:
added